README.md

```diff
@@ -17,6 +17,15 @@ with the following differences:
   * Provides the fallback list when using tools like `browserify` without pulling
     in the `http` shim module.
 
+> [!NOTE]
+> This module is largely a wrapper of [`http.METHODS`](https://nodejs.org/api/http.html#httpmethods) from Node.js core with backwards compatibility for Node.js versions older than 0.11.8, making it mostly unnecessary for use in new projects. For newer Node.js versions, the following code is equivalent to using this module:
```

Comment:
In npm, it doesn't follow the same format, and the styles that GitHub applies to this type of notes wouldn't be applied

Reply:
Bummer, but would that be a reason to hold it up? I believe NPM will just show this as quoted text, which seems a reasonable fallback.

Reply:
Let's see if we can clarify this in the npm channel: https://openjs-foundation.slack.com/archives/CTEDKHCTB/p1729968334939499 :thinking:

Reply:
The view on npm would look something like this

![image](https://github.com/user-attachments/assets/488a4b12-dc83-4897-a4a1-381bc0de230c)

Reply:
Doesn’t it basically say this above already? Why not reword the exist text saying what the package is for? Existing text:

> This module provides an export that is just like `http.METHODS` from Node.js core,
with the following differences
> […] * Contains a fallback list of methods for Node.js versions that do not have a `http.METHODS` export (0.10 and lower).

Can we try to merge the redundancy in this description? Maybe link to the node docs instead of the code sample since you can write it at least 4 different ways nowadays depending on CommonJS, ESM, non-node prefixes?

Reply:
I've revised the notice not to repeat aforementioned statements and simply instruct the user to use the core module if their support target allows, sans code example.